README.md

```diff
@@ -56,18 +56,24 @@ Files in .gitignore (auto-generated)
 For functions not yet implemented:
 
 ```js
-function function_name()) {
+function functionName()) {
     throw new Error(`TODO: Not implemented`);
 }
 ```
 
-And for functions that will be not implemented:
+And for functions or classes that will not be implemented:
```

Comment:
Are all (at present moment and in the future) unimplemented entities classes or function? Are there some other entities?

Reply:
I deleted the entity definition. It can be derived from the naming convention in JS (camelCase for functions and PascalCase for classes).